qa-portal-angular/projects/qa-cv/src/app/view-cv/services/view-cv-page-data.service.ts

```diff
@@ -1,51 +1,50 @@
 import {Injectable} from '@angular/core';
-import {UserSkillsModel} from '../../_common/models/user-skills.model';
-import {DEFAULT_CV, ICvModel} from '../../_common/models/qac-cv-db.model';
-import {IN_PROGRESS_STATUS} from '../models/view-cv.constants';
+
+import {IN_PROGRESS_STATUS} from '../../_common/models/cv-status.constants';
 
 @Injectable()
 export class ViewCvPageDataService {
   constructor() {
   }
 
-  public initialiseNewCv(cvData: ICvModel, useExistingCvAsTemplate: boolean): ICvModel {
-    const tempCv = cvData;
-    const emptyCv = null;
-    let newCv = null;
-    if (!!useExistingCvAsTemplate) {
-      newCv = {...DEFAULT_CV, ...cvData};
-      newCv.id = null;
-      newCv.versionNumber = null;
-    } else {
-      newCv = {...DEFAULT_CV, ...emptyCv};
-      newCv.allSkills = tempCv.allSkills;
-      newCv.fullName = tempCv.fullName;
-    }
-    newCv.status = IN_PROGRESS_STATUS;
-    return newCv;
-  }
+  // public initialiseNewCv(cvData: ICvModel, useExistingCvAsTemplate: boolean): ICvModel {
```

Comment:
Cleanup this file. There's a lot of commented out code.

Reply:
qa-portal-angular/projects/qa-cv/src/app/view-cv/services/view-cv.service.ts
is deleted on this branch as it is not needed. 
I don't understand why this has appeared in the merge.